Sources/Starknet/Accounts/StarknetAccount.swift

```diff
@@ -30,44 +24,67 @@ public class StarknetAccount: StarknetAccountProtocol {
         self.cairoVersion = cairoVersion
     }
 
-    private func makeInvokeTransaction(calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetExecutionParams, forFeeEstimation: Bool = false) -> StarknetInvokeTransactionV1 {
+    private func makeInvokeTransactionV1(calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetDeprecatedExecutionParams, forFeeEstimation: Bool = false) -> StarknetInvokeTransactionV1 {
         StarknetInvokeTransactionV1(senderAddress: address, calldata: calldata, signature: signature, maxFee: params.maxFee, nonce: params.nonce, forFeeEstimation: forFeeEstimation)
     }
 
-    private func makeDeployAccountTransaction(classHash: Felt, salt: Felt, calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetExecutionParams) -> StarknetDeployAccountTransactionV1 {
-        StarknetDeployAccountTransactionV1(
-            signature: signature,
-            maxFee: params.maxFee,
-            nonce: params.nonce,
-            contractAddressSalt: salt,
-            constructorCalldata: calldata,
-            classHash: classHash
-        )
+    private func makeInvokeTransactionV3(calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetExecutionParamsV3, forFeeEstimation: Bool = false) -> StarknetInvokeTransactionV3 {
```

Comment:
The reasoning was:
1. `StarknetDeprecatedExecutionParams` - both for v1 and v2s, `v1` postfix won't cut it.
2. `StarknetExecutionParamsV3 ` - for v3s. 

Didn't want to remove the `V3` postfix because these are still the new thing. Now to think of it, there is no support for sending declares in swift, so we could go for `StarknetExecutionParamsV1` 😄

Reply:
I get the rationale, but I probably wouldn't bother and just use `DeprecatedStarknetExecutionParams` (just to make Depracated pop out more, but your name is also fine) and `StarknetExecutionParams` 😅

Reply:
> and StarknetExecutionParams

Still unsure about this change. 

It makes sense that we pass `InvokeParamsV3` to `signV3`. It doesn't, however, make much sense why we can't pass something called `InvokeParams` to `signV1`.